### PR TITLE
Makes heliostation AI sat awesome

### PR DESCRIPTION
**Pull request by Pepsilawn**
Helio AI sat used to look like this

image

Helianthus in his folly somehow managed to run out of areas to use and ended up using the parent type which looks really bad.
Also I don't know how many people when you tell them "Lesser starboard maints" will immediately think of plasmaman medbay

I spent 2 entire minutes in dream maker

image

### CODE REVIEW

File: fulp_modules/mapping/areas/areas.dm [hunk 1]
```diff
@@ -5,9 +5,28 @@
...
  */
 
+// Station
+
 /area/service/lawoffice/upper
 	name = "\improper Upper Law Office"
 
+/area/station/ai_monitored/turret_protected/aisat/solars
+	name = "\improper AI Satellite Solars"
+	icon = "fulp_modules/mapping/areas/icons.dmi"
```
Comment (JohnFulpWillard) [inline fulp_modules/mapping/areas/areas.dm:15]:
```suggestion
	icon = 'fulp_modules/mapping/areas/icons.dmi'
```

File: fulp_modules/mapping/areas/areas.dm [hunk 2]
```diff
@@ -5,9 +5,28 @@
...
+	icon_state = "ai_solars"
+
+/area/station/solars/ai
+	name = "\improper AI Satellite Solar Array"
+	icon = "fulp_modules/mapping/areas/icons.dmi"
```
Comment (JohnFulpWillard) [inline fulp_modules/mapping/areas/areas.dm:20]:
```suggestion
	icon = 'fulp_modules/mapping/areas/icons.dmi'
```

File: fulp_modules/mapping/areas/areas.dm [hunk 3]
```diff
@@ -5,9 +5,28 @@
...
+	icon_state = "ai_panels"
+
+/area/station/maintenance/department/medical/plasmaman
+	name = "\improper Plasmaman Medbay"
+	icon = "fulp_modules/mapping/areas/icons.dmi"
```
Comment (JohnFulpWillard) [inline fulp_modules/mapping/areas/areas.dm:25]:
```suggestion
	icon = 'fulp_modules/mapping/areas/icons.dmi'
```

**Comment by Pepsilawn** [comment]
thank you john willard for helping me make my first pr

**Pull request merged**